<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Visualization</class>
 <widget class="QMainWindow" name="Visualization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualization App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="equationsTxt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>481</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="equationsLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Equations</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="outputTxt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>721</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="outputLbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Program Output</string>
    </property>
   </widget>
   <widget class="QPushButton" name="solve">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>290</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Solve</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="readFromFile">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Read system of equations from file</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="fromPoints">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Construct system of equations from points</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>select file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="InsertPoints">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert Points 1</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="pointsTxt">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>150</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>130</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Points</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="d1Box">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>100</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="d2Box">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>100</y>
      <width>48</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>100</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>d1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>100</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>d2</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="points2Txt">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>150</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="InsertPoints2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>60</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert Points 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>130</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Points 2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>20</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>true</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Alt+F</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
